Directory.Packages.props

```diff
@@ -98,8 +98,8 @@
     <PackageVersion Include="Humanizer.Core" Version="2.14.1" />
     <PackageVersion Include="KubernetesClient" Version="13.0.26" />
     <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.2.0" />
-    <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components" Version="4.6.1" />
-    <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components.Icons" Version="4.6.1" />
+    <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components" Version="4.7.0" />
```

Comment:
Looks like one is already 4.7.1?

Reply:
Yes, found an issue in 4.7.0 and published new NuGet couple of hours later. PR now updated for this